docs/manage/data/view.md

```diff
@@ -44,6 +44,6 @@ You can also delete data using the [Viam CLI](../../cli/).
 
 To export your captured data from the cloud, see [Export Data](../export/).
 
-You can use cloud image data to [train machine learning models](../../ml/train-model/) within Viam.
+You can [query](../query/) tabular data that you have synced to the cloud, and can use images you have synced to [train machine learning models](../../ml/train-model/) within the Viam app.
```

Comment:
Similar to other comments, I want to make sure users know they can filter all binary/tabular data using the Python APIs. In the above `Filter data` section, the UI is actually just using those BinaryDataByFilter/TabularDataByFilter APIs in the backend. This can be captured in the `Filter data` section and perhaps just 'flexibly query' here to make the distinction clear?